### PR TITLE
Setup/navbar

### PR DESCRIPTION
**Pull request by KonstancjaMG**

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { renderApp } from "./render.js"
+import { LINKS } from "./data.js"
+
+const MAIN_HTML = document.getElementById("app")
```
Comment (hi-coders) [inline js/main.js]: This is not a constant, write it with camelcase format.
Use container as suffix instead of "html"

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { renderApp } from "./render.js"
+import { LINKS } from "./data.js"
```
Comment (hi-coders) [inline js/main.js]: links is also a dynamic structure which could be loaded from tha backend, so it is not a constant.

**Review by hi-coders — CHANGES_REQUESTED** [review]
It seems ok in this phase.
Later on I will request some changes in your approach!

**Comment by KonstancjaMG** [comment]
@hi-coders > It seems ok in this phase. Later on I will request some changes in your approach!

Should I redo this with templates? We just had mentor hours with Cemil Dogan and he showed us how to use them. He said that the menu won't be changing much so it doesn't need to be dynamic from an array.

**Pull request merged**